iree/tools/android/run_module_app/build_apk.sh

```diff
@@ -32,67 +32,41 @@ echo "===-------------------------------------------==="
 echo ""
 
 print_usage_and_exit() {
-  echo "Usage: $0 <artifact-directory> "
-  echo "       --driver <driver>"
-  echo "       --module_file <input-module-file> "
-  echo "       --entry_function <entry-function> "
-  echo "       --inputs_file <input-buffer-file> "
+  echo "Usage: $0"
+  echo "       -d <driver>"
+  echo "       -m <input-module-file> "
+  echo "       -e <entry-function> "
+  echo "       -i <input-buffer-file> "
```

Comment:
Single letter flags? Not very easy to remember or parse from command history. Can we use full names and abbreviations?

Reply:
AFAIK, getopts does not support multiple-character options. That's why the previous impl did not use getopts. I prefer the original impl too. Switching it back is just reverting 9619a04. But I'm fine either way given this improves the script a bit.

Reply:
Yeah getopts doesn't. *getopt* does, but has some other issues. There appears to be some kind of holy war about it. I'm fine with the original manual parsing, although it's pretty sad that bash doesn't have a good solution for command line parsing :-/

Reply:
My preference would be for multiple-character options, even if the parsing code is messier.

Reply:
SG. Reverted to the original impl.